### PR TITLE
Feature:elevator (v1)

### PR DESCRIPTION
**Pull request by duane701**
Rebased from #2

### CODE REVIEW

File: src/Commands/SetElevatorMove.cpp [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+// Make this return true when this Command no longer needs to run execute()
+bool SetElevatorMove::IsFinished() {
+	if (Elevator::getInstance()->IsFwdLimitSwitchClosed() || Elevator::getInstance()->IsRevLimitSwitchClosed()) {
+		return true;
+	}
+	else {
+		return false;
+	}
+}
```
Comment (duane701) [inline src/Commands/SetElevatorMove.cpp]: Simplify this logic!

File: src/Commands/SetElevatorPos.cpp [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+void SetElevatorPos::Execute() {
+	Elevator::getInstance()->setElevatorPos(mPosition);
+}
+
+// Make this return true when this Command no longer needs to run execute()
+bool SetElevatorPos::IsFinished() {
+	if (Elevator::getInstance()->GetPosError() < 1000
+			|| Elevator::getInstance()->IsFwdLimitSwitchClosed()
+			|| Elevator::getInstance()->IsRevLimitSwitchClosed()) {
+		if (counter > 29) {
```
Comment (duane701) [inline src/Commands/SetElevatorPos.cpp]: Need a name for this magic number.

File: src/OI.cpp [hunk 3]
```diff
@@ -9,7 +9,7 @@
 // it from being updated in the future.
 
 
-#include <Commands/ElevatorDown.h>
+#include <Commands/SetElevatorPos.h>
```
Comment (duane701) [inline src/OI.cpp]: Why is `#include <Commands/SetElevatorPos.h>` way up here?

**Review by duane701 — COMMENTED** [review]
Grr, I started this PR so I can't approve or request changes.

**Review by MajorDarius — APPROVED** [review]
LGTM

**Pull request merged**